zad1/main.rb

```diff
@@ -1,21 +1,39 @@
-def sort_letters(string)
-
+def sort_letters(str)
+  i=0
+  while i<(str.length-1)
```

Comment:
W rubym raczej unikamy pętli `while`. Załóżmy, że mamy array ze znakami `chars = ['a', 'b', 'c']`. Jeśli chcemy zrobić coś z każdym elementem tego array nie używamy licznika, `while` ani `for` ale zamiast tego metod typu `each`, `map` itd.
[TUTAJ](http://jeromedalbert.com/ruby-how-to-iterate-the-right-way/) możesz poczytać o tym jak dobrze iterować w rubym 😃 

W tym konkretnym przypadku nie musisz implementować algorytmu sortowania(oczywiście dobrze jest je znać) - [rubiowa klasa Array](https://ruby-doc.org/core-2.5.3/Array.html) kryje w sobie bardzo dogodny sposób sortowania 😉